### PR TITLE
Add more aurora metrics to export

### PR DESCRIPTION
**Pull request by jbripley**
Add helper method to create metric description to cut down on boilerplate.

### CODE REVIEW

File: vars.go [hunk 1]
```diff
@@ -8,274 +8,470 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+func newMetric(subsys string, name string, descr string) *prometheus.Desc {
```
Comment (tommyulfsparre) [inline vars.go]: newMetric -> newDesc ? 

If the argument is of the same type you only need to specify it once: 
`subsys, name, descr string`

Comment (jbripley) [inline vars.go]: Yep, newDesc is a more descriptive name.

Ah, didn't know that about function definitions. I'll change it.

**Comment by StephanErb** [comment]
There are a few metrics which seem to be in the wrong category. To prevent merge conflicts it would probably be best if you change them in this pull request.

Should be counter:
- `jvm_class_loaded_count`
- `jvm_class_unloaded_count` 
- `jvm_memory_non_heap_mb_max`

Should be gauge:
- `schedule_queue_size`
- `pubsub_executor_queue_size`

**Comment by tommyulfsparre** [comment]
@jbripley Beautiful, thx!

**Pull request merged**